src/v3/src/components/AuthCoin/AuthCoinConfig.ts

```diff
@@ -154,6 +154,13 @@ const AuthCoinByAuthenticatorKey: Record<string, AuthCoinConfig> = {
     description: loc('factor.totpHard.yubikey', 'login'),
     iconClassName: 'mfa-yubikey',
   },
+  [CHALLENGE_METHOD.CUSTOM_URI]: {
+    icon: OktaVerifyIcon,
+    name: 'mfa-okta-verify',
+    customizable: false,
+    description: loc('factor.totpSoft.oktaVerify', 'login'),
+    iconClassName: 'mfa-okta-verify',
+  },
```

Comment:
nit: Can you move this up under `[AUTHENTICATOR_KEY.CUSTOM_OTP]: { ... }` since it looks like these objects are ordered alphabetically by key?

Reply:
I was trying to decide if I should group these by the first part of the constant name (e.g. AUTHENTICATOR_KEY, CHALLENGE_METHOD) and then alphabetical within those groups, or just alphabetical across all. What do you think?

Reply:
Ohhh, sorry, my eyes are playing tricks on me, I think here is fine 😅 since this is ordered by the first part of the constant. Sorry about that my brain totally ignored the `CHALLENGE_METHOD` part 😅